packages/financial-templates-lib/src/price-feed/PriceFeedInterface.js

```diff
@@ -21,7 +21,7 @@ class PriceFeedInterface {
   // return `null` or `undefined`. If the historical price could not be computed for any other reason, this method
   // should return `null` or `undefined`.
   // Note: derived classes *must* override this method.
-  getHistoricalPrice(time) {
+  getHistoricalPrice(/* time */) {
```

Comment:
So `time` is not part of the interface? why not just remove it

Reply:
It is part of the interface, it just isn't used because this is the interface _definition_. If you look at other classes that inherit from it, they use `time`.